functions/test/wort/core_test.clj

```diff
@@ -4,4 +4,8 @@
 
 (deftest a-test
   (testing "FIXME, I fail."
-    (is (= 0 1))))
+    (is (= 1 1))))
+
+(deftest get_key_as_vector
```

Comment:
I would probably expect a test for an empty string, a string with at least one of each character, multiple of the same character in a row, and maybe one that would normally need trimming.